src/samples/Uno.Themes.Samples/Uno.Themes.Samples.Shared/Content/Controls/ButtonSamplePage.xaml

```diff
@@ -18,47 +18,74 @@
-						Style="{StaticResource MaterialContainedButtonStyle}" />
+									Style="{StaticResource MaterialContainedButtonStyle}" />
 						</smtx:XamlDisplay>
 
 						<!-- Button Contained Disabled -->
 						<smtx:XamlDisplay UniqueKey="ButtonSamplePage_ContainedDisabled">
 							<Button Content="CONTAINED DISABLED"
-						IsEnabled="False"
-						Style="{StaticResource MaterialContainedButtonStyle}" />
+									IsEnabled="False"
+									Style="{StaticResource MaterialContainedButtonStyle}" />
 						</smtx:XamlDisplay>
 
 						<!-- Button Outlined -->
 						<smtx:XamlDisplay UniqueKey="ButtonSamplePage_Outline">
 							<Button Content="OUTLINED"
-						Style="{StaticResource MaterialOutlinedButtonStyle}" />
+									Style="{StaticResource MaterialOutlinedButtonStyle}" />
 						</smtx:XamlDisplay>
 
 						<!-- Button Outlined Disabled -->
 						<smtx:XamlDisplay UniqueKey="ButtonSamplePage_OutlineDisabled">
 							<Button Content="OUTLINED DISABLED"
-						IsEnabled="False"
-						Style="{StaticResource MaterialOutlinedButtonStyle}" />
+									IsEnabled="False"
+									Style="{StaticResource MaterialOutlinedButtonStyle}" />
 						</smtx:XamlDisplay>
 
 						<!-- Button Text -->
 						<smtx:XamlDisplay UniqueKey="ButtonSamplePage_Text">
 							<Button Content="TEXT"
-						Style="{StaticResource MaterialTextButtonStyle}" />
+									Style="{StaticResource MaterialTextButtonStyle}" />
 						</smtx:XamlDisplay>
 
 						<!-- Button Text Disabled -->
 						<smtx:XamlDisplay UniqueKey="ButtonSamplePage_TextDisabled">
 							<Button Content="TEXT DISABLED"
-						IsEnabled="False"
-						Style="{StaticResource MaterialTextButtonStyle}" />
+									IsEnabled="False"
+									Style="{StaticResource MaterialTextButtonStyle}" />
 						</smtx:XamlDisplay>
 					</StackPanel>
 				</DataTemplate>
 			</sample:SamplePageLayout.MaterialTemplate>
 			<sample:SamplePageLayout.CupertinoTemplate>
 				<DataTemplate>
 					<StackPanel>
+						<!-- Button Default -->
+						<Button Content="DEFAULT"
+								Style="{StaticResource CupertinoDefaultButtonStyle}"
+								Margin="5"/>
+						<!-- Button Default Disabled -->
+						<Button Content="DISABLED"
+								IsEnabled="False"
+								Style="{StaticResource CupertinoDefaultButtonStyle}"
+								Margin="5" />
+						<!-- Button Contained -->
+						<Button Content="CONTAINED"
+								Style="{StaticResource CupertinoContainedButtonStyle}"
+								Margin="5" />
+						<!-- Button Contained Disabled-->
+						<Button Content="DISABLED"
+								IsEnabled="False"
+								Style="{StaticResource CupertinoContainedButtonStyle}"
+								Margin="5" />
+						<!-- Button Outlined -->
+						<Button Content="OUTLINED"
+								Style="{StaticResource CupertinoOutlinedButtonStyle}"
+								Margin="5" />
+						<!-- Button Outlined Disabled-->
+						<Button Content="DISABLED"
+								IsEnabled="False"
+								Style="{StaticResource CupertinoOutlinedButtonStyle}"
+								Margin="5" />
```

Comment:
We should be wrapping each button in the smtx:XamlDisplay element (with a unique key)